renovate.json

```diff
@@ -112,6 +112,22 @@
         "patch",
         "renovate-deps"
       ]
+    },
+    {
+      "depTypeList":[
+        "devDependencies"
+      ],
+      "matchUpdateTypes":[
+        "patch"
+      ],
+      "matchPackagePrefixes":[
+        "@types"
+      ],
+      "labels":[
+        "no release",
+        "patch"
```

Comment:
We probably don't want `patch` label on here. It conflicts with `no release` for our release/build workflows